redmine/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+
+services:
+
+  redmine:
+    image: redmine
+    ports:
+      - 8080:80
```

Comment:
Uhhh, I think this should be 3000: https://github.com/docker-library/redmine/blob/48c809da106963f2f373ba0250a03e9b6daa8bd3/Dockerfile.template#L89